### PR TITLE
Add light nomad gear

### LINKED ISSUE #27336: Lower Warmth value of Nomad Gear.
Nomad gear used to be one of my favourite clothing items in the early-to-mid game, due it's it's excellent combination of good storage space and reasonable encumbrance and protection. Unfortunately, back in 2015 Marrim adjusted it, increasing the Warmth rating of the armor to 40, meaning that it's simply too hot to use consistently in the majority of seasons and weather conditions, which in my opinion goes against the entire theme of the item, and makes what was previously a great early game staple irritating to manage. A nomad would not design his primary outerware such that it was unusable for 3/4 of the year. That's what long underwear is for.

I suggest that the item's warmth be reduced to 20, which is the average warmth of the clothing that goes into crafting it.

### PR DESCRIPTION
**Pull request by Pupsi-Mupsi**
#### Summary

`SUMMARY: Content "Add light nomad gear"`

#### Purpose of change

As mentioned by @Solusphere and supported by a few others there seems to be the wish for a **light summary version** of the established **nomad gear**. The light version is based solely on a simple **tunic** and therefore  offers less protection and storage. Just use the existing **turban** to cover your head

- Please let me know what you think of it and if it should be changed in any way.
- @Marrim suggested in #27336 "_Might want to change it away from normal layer, though._"
-> Following his suggestion the regular nomad gear should then be moved to the outer layer as well, should it?

**Update:**
- Thanks to @GoLoT -> Add weight, shorts cargo (or shorts denim), and slightly decrease crafting-difficulty
- Removed waterproof (as Rivet-the-Zombie pointed out).

<br />
Closes #27336.
<br /><br />

New light version:
grafik

Old nomad gear:
grafik

Reinforced with leather:
grafik

Reinforced with kevlar:
grafik

With fur and wool lining:
grafik

### CODE REVIEW

**Comment by GoLoT** [comment]
I think the cargo/army pants requirement should be kept, it's the only part of the recipe that has to be looted or crafted from a book recipe. Removing it makes the light version much easier to craft than the regular one.

Not using any leather also makes it much easier to craft, maybe it should be 20 rags + 12 leather patches or something like that. Other than that I think it's a good approach to solve the issue.

**Comment by Pupsi-Mupsi** [comment]
> 
> 
> I think the cargo/army pants requirement should be kept, it's the only part of the recipe that has to be looted or crafted from a book recipe. Removing it makes the light version much easier to craft than the regular one.
> 
> Not using any leather also makes it much easier to craft, maybe it should be 20 rags + 12 leather patches or something like that. Other than that I think it's a good approach to solve the issue.

Thanks. I was hoping for more feedback.
How about increasing the crafting difficulty to 6 instead? So the recipe can stay light and less protective.

**Comment by GoLoT** [comment]
Considering light armor of other tiers is easier to craft than the heavier versions I would lower the difficulty instead to make it consistent.

As an alternative to using cargo or army pants maybe you could make it require cargo/denim shorts instead. Both can be crafted from the same book as cargo pants or as a regular recipe if you have cargo pants or jeans.

**Comment by Pupsi-Mupsi** [comment]
> 
> 
> Considering light armor of other tiers is easier to craft than the heavier versions I would lower the difficulty instead to make it consistent.
> 
> As an alternative to using cargo or army pants maybe you could make it require cargo/denim shorts instead. Both can be crafted from the same book as cargo pants or as a regular recipe if you have cargo pants or jeans.

Thanks! Changed the recipe and item according to your suggestion.

**Comment by Siersha** [comment]
80% coverage is very harsh considering armor balance essentially making the clothing article only block 45% of the time. (more info here https://github.com/CleverRaven/Cataclysm-DDA/wiki/Armor-balance). 95% coverage would make the time and effort into making it more worthwhile. Otherwise, I would simply spend the extra day or two to make the nomad gear (which can be built very easily - the only "un-craftable" thing to get from it are cargo pants. Just my 2c.

**Comment by kevingranade** [comment]
> 80% coverage is very harsh considering armor balance essentially making the clothing article only block 45% of the time.

That's not how it works, what you are quoting is a formula to evaluate effectiveness of armor.  80% coverage blocks 80% of attacks.

File: data/json/items/armor.json [hunk 1]
```diff
@@ -518,6 +518,28 @@
...
+    "symbol": "[",
+    "color": "light_gray",
+    "covers": [ "TORSO", "ARMS", "LEGS" ],
+    "coverage": 80,
+    "encumbrance": 16,
+    "storage": 50,
+    "warmth": 10,
+    "material_thickness": 4,
+    "environmental_protection": 1,
+    "flags": [ "VARSIZE", "POCKETS", "WATERPROOF" ]
```
[the remark this answers is not in the document]
Comment (Toothspit) [reply data/json/items/armor.json]: Old nomad gear is waterproof as well, with no waterproof ingredients in the recipe. Not sure if that was a justified abstraction or hand-waivey 'better than a hoodie!' thing at the time. Might be worth another PR to rebalance that.

[the remark this answers is not in the document]
Comment (Toothspit) [reply data/json/items/armor.json]: And now I've sat here for 10 minutes and thought about whether waterproofing should be a cool 'nother use of the washboard, a tailoring thing, add a Scotch-guard item, or rub-some-beeswax--or-turpentine-on-it.

No clue from a gameplay perspective. Leaving this one to greater minds.

[the remark this answers is not in the document]
Comment (Pupsi-Mupsi) [reply data/json/items/armor.json]: Thanks! Removed the WATERPROOF flag from both nomad gears.

**Pull request merged**